### PR TITLE
Update tvtv.us_us.channels.xml

### PR DESCRIPTION
**Pull request by Arkina1234**

### CODE REVIEW

**Comment by Central302** [comment]
WWOR and WNYE is Already Added,

**Comment by Central302** [comment]
Plese Ignoire This Mentioning (Conflict Will Fix Myself)

**Review by Arkina1234 — COMMENTED** [review]
update

**Review by Central302 — APPROVED** [review]

File: sites/tvtv.us/tvtv.us_us.channels.xml [hunk 1]
```diff
@@ -1395,6 +1395,30 @@
     <channel lang="en" xmltv_id="WDPNDT3.us" site_id="90140">Court TV Mystery (WDPN-DT3) Philadelphia PA</channel>
     <channel lang="en" xmltv_id="WDPNDT4.us" site_id="91607">Heroes and Icons (WDPN-DT4) Philadelphia PA</channel>
     <channel lang="en" xmltv_id="WDPNDT5.us" site_id="106849">Retro TV (WDPN-DT5) Philadelphia PA</channel>
+    <channel lang="en" xmltv_id="WNETDT1.us" site_id="26182">PBS (WNET-DT1) New York, NY</channel>
```
Comment (Arkina1234) [inline sites/tvtv.us/tvtv.us_us.channels.xml:1398]: F

**Review by Arkina1234 — COMMENTED** [review]
requested

**Review by Central302 — APPROVED** [review]

**Review by freearhey — APPROVED** [review]

**Pull request merged**